kmm/kmm-post-data/src/commonMain/kotlin/com/gchristov/newsfeed/kmmpostdata/RealPostRepository.kt

```diff
@@ -53,7 +38,7 @@ internal class RealPostRepository(
             }
         }
 
-    private fun cachePost(decoratedPost: DecoratedPost) {
+    override suspend fun cachePost(decoratedPost: DecoratedPost) {
```

Comment:
As before, this can be caching just a `Post` for now and we'll address in a separate PR.

Reply:
Discussed above, the change seems too invasive in signatures and tests